site/application_wrapper.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 if (typeof window !== 'undefined') {
-  import(/* webpackChunkName: "assembly-js" */ '../dist/assembly.js');
+  import(/* webpackChunkName: "assembly-js" */ '../dist/assembly.js'); // eslint-disable-line import/no-unresolved
```

Comment:
To travis, this file doesn't exist since Travis does not build assembly. Alternatively, we could add `npm run build` to travis's scripts, but that would add extra time to each run.